doc/specs/#605 - Search/spec.md

```diff
@@ -9,114 +9,114 @@ issue id: #605
+  - The two buttons with up/down arrows controls the search direction, Each button will be styled to indicate which search direction is currently selected.
+  - The button with a "Aa" icon, if pressed, means that we are searching case-sensitivity.
+  - The current style puts all elements - the `X` button, the text box and the search pattern control buttons on one single line. This ensures that the `SearchBoxControl` won't be too high and block terminal text. This is similar with VSCode. Another possible layout style is to put elements in multiple layers. This will occupy more lines, but the search dialog will narrower. Considering that there is not many elements, we do not need multiple layers.
 
 ![SearchBox mockup, arrow button clicked](images/SearchBoxUpSelected.png)
 
-The search box defaults to be on the top right corner of the Terminal window. If the current tab is split into panes, each pane will have a individual searchbox. 
+The search box defaults to be on the top right corner of the Terminal window. If the current tab is split into panes, each pane will have a individual searchbox.
 
 #### Search process
-1. The user presses <kbd>ctrl+shift+f</kbd> (or user's custom key binding) to open the search box. Focus will move to the TextBox. 
-2. Search is performed on a XAML TextBox. Once the user presses Enter or click up/down arrow button, the search starts and searched text will be selected. Next search will be performed beginning from the current selection and go towards up/down. 
-3. The user can choose to search up or down by selecting up arrow or down arrow buttons. The chosen button will be styled to indicate it is selected. If the user does not click the arrows buttons, the default direction is up. 
-4. The user can choose to do case sensitive or insensitive match by checking a check box. The default is case insensitive. 
-5. If the search box is focused, the user can navigate all the elements on the search box using tab. When selected, press Enter equals to click. 
+1. The user presses <kbd>ctrl+shift+f</kbd> (or user's custom key binding) to open the search box. Focus will move to the TextBox.
+2. Search is performed on a XAML TextBox. Once the user presses Enter or click up/down arrow button, the search starts and searched text will be selected. Next search will be performed beginning from the current selection and go towards up/down.
+3. The user can choose to search up or down by selecting up arrow or down arrow buttons. The chosen button will be styled to indicate it is selected. If the user does not click the arrows buttons, the default direction is up.
+4. The user can choose to do case-sensitive or insensitive match by checking a check box. The default is case-insensitive.
+5. If the search box is focused, the user can navigate all the elements on the search box using tab. When selected, pressing Enter is equivalent to to click.

```

Comment:
This is trying to address that `equals to`. Here the fix was more complicated than an obvious search+replace.

Reply:
Oops. Fwiw, the warning only triggers for words of at least 3 characters -- see https://github.com/microsoft/terminal/pull/18261/files#r1868853584